src/libpanic_unwind/seh.rs

```diff
@@ -208,6 +208,17 @@ static mut TYPE_DESCRIPTOR: _TypeDescriptor = _TypeDescriptor {
     name: TYPE_NAME,
 };
 
+/// Destructor used if the C++ code decides to capture the exception and drop it
+/// without propagating it.
+#[cfg(target_arch = "x86")]
+unsafe extern "thiscall" fn exception_cleanup(e: *mut [u64; 2]) {
```

Comment:
Could you add a documentation comment or something about why we're using thiscall specifically for x86? (i.e., a link to the spec or something)